app/tests/spec/views/connect_another_device.js

```diff
@@ -306,130 +306,27 @@ define(function (require, exports, module) {
       it('returns `false` if user is signed in', () => {
 
         sinon.stub(user, 'isSignedInAccount', () => true);
-        sinon.stub(view, '_hasWebChannelSupport', () => true);
+        sinon.stub(view, '_webChannelSupportExists', () => true);
```

Comment:
if [_webChannelSupportExists is renamed to _hasWebChannelSupport](https://github.com/mozilla/fxa-content-server/pull/4798/files#r109491117), these stub's won't need to be updated.